chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.JoinedRoom
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class JoinedRoomTest : Spek({
+
+    describe("given an empty joined room state") {
+        val initialState = ChatkitState(
+                joinedRoomsState = JoinedRoomsState(mapOf(), mapOf()))
+
+        describe("when one new room with unread counts is received") {
+            val joinedRoom = JoinedRoom(JoinedRoomsStateTestUtil.roomOne, unreadCount = 1)
+            val newState = joinedRoomReducer(initialState, joinedRoom)
+
+            it("then the state should contain the expected rooms") {
```

Comment:
```suggestion
            it("then the state should contain the expected room") {
```

Reply:
👍 the auto commit didn't seem to work so manually added it.